### DISCUSSION TITLE
过期：懒人包&简易包 20220515

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#124] [回收]
## 版本专帖

该版起打包改用自解压格式

<hr /><em>link <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20220515'>mpv-lazy-20220515</a></em>

<hr /><em>link <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/5900'>mpvnet-CM-20220515</a></em>

- **20220606** 追加6000CM的可选版本更新（补丁） https://github.com/hooke007/mpv.net_CM/releases/tag/6000

**Comment by hu3rror** [comment]
自用的 Scoop 包已同步自解压格式的更新😆： https://github.com/hu3rror/scoop-muggle/commit/79e9b09e6528038fd3ef4201cb3c61409e5a43d9

**Comment by hooke007** [comment]
6000CM的补丁已追加，以后正式的整合版会随每一个懒人包的版本一起更新，想尝鲜的使用补丁即可。

**Reply by ic2391** [reply]
视频输出驱动改为gpu-next后，窗口播放字幕能正常显示，全屏不显示字幕或只显示一半。

![Snipaste_2022-06-24_13-12-59](https://user-images.githubusercontent.com/62416821/175468316-bfa94989-698c-499f-baf4-d1696d710cc3.png)
![Snipaste_2022-06-24_13-14-36](https://user-images.githubusercontent.com/62416821/175468334-1539616e-a85c-4532-b104-5896eb160e80.png)
![Snipaste_2022-06-24_13-16-16](https://user-images.githubusercontent.com/62416821/175468343-f78280a1-001f-45ab-a9fc-0baf3ab9e31c.png)

**Reply by hooke007** [reply]
这是gpu-next的已知问题

**Comment by linger040831** [comment]
![image](https://user-images.githubusercontent.com/84896599/179397357-f6e6298c-2a24-403b-b5bb-1fb0929db67d.png)
当idle为no时会报错，已设置重置，mpv.conf只有这一条参数，mpvl-lazy里这个设置不会出现这种情况，请问是不是什么原因？

**Reply by hooke007** [reply]
https://github.com/stax76/mpv.net/blob/master/docs/Manual.md#other-limitations

**Reply by linger040831** [reply]
好的知道原因了，谢谢回答！

**Comment by M1saka10010** [comment]
差不多该再发个release了吧大佬
